### DISCUSSION TITLE
windows11

### DISCUSSION THREAD

**Discussion by Banxiaxiala** [File-New-Project/EarTrumpet#975] [General]
How to go back to the old version of the volume synthesizer
![image](https://user-images.githubusercontent.com/35516655/146320904-d7dcb87f-8644-452a-a04e-823531954415.png)

**Comment by riverar** [comment]
@1299897460 This looks like a bug. What version of EarTrumpet are you on? What version of Windows?

**Reply by Banxiaxiala** [reply]
版本	Windows 11 专业工作站版
版本	21H2
安装日期	‎2021/‎9/‎14
操作系统版本	22000.376
体验	Windows 功能体验包 1000.22000.376.0
EarTrumpet The latest version

**Reply by ShreyanshTech** [reply]
https://youtu.be/s_v2Ury0D6c
 see this walkthrough
